Makefile

```diff
@@ -0,0 +1,29 @@
+current_dir = $(shell pwd)
+# uid = $(shell id -u)
+# gid = $(shell id -g)
+
+define molecule_docker_exec 
+docker run \
+-it --rm \
+-v "/var/run/docker.sock:/var/run/docker.sock" \
+-v "$(current_dir):$(current_dir)" \
+-w "$(current_dir)" \
+--entrypoint="" \
+-e "HOME=/tmp/" \
```

Comment:
```suggestion
```
are these necessary

Reply:
Nah they can be dropped. HOME is cruft left over from the UID fun && the entrypoint for the image is already set to sh